Makefile

```diff
@@ -1,6 +1,6 @@
 DEV_CONTAINER_TAG:=devcontainer
 
-build: 
+build: checks
```

Comment:
`checks` doesn't exist anymore

Reply:
Oh man - good spot - trying to do too much and not focusing.